stable/prometheus/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 5.5.1
+version: 6.0.0
```

Comment:
Not sure why you bumped the major, this doesn't look to me like a backwards incompatible change?

Reply:
@timstoop I'm happy to change it to a minor, but this might change the name of a PVC for instance I think, resulting in a pod starting with a new (empty) PV instead of the PV already in place.

Reply:
Ah that makes sense, then. Let's leave it at a major version bump.